ui-components/src/forms/HouseholdMemberForm.tsx

```diff
@@ -2,11 +2,18 @@ import React from "react"
 import { useRouter } from "next/router"
 import { HouseholdMemberUpdate } from "@bloom-housing/backend-core/types"
 import { t } from "../helpers/translator"
+import { AppearanceStyleType } from "../global/AppearanceTypes"
+import { Icon } from "../icons/Icon"
 import { ViewItem } from "../blocks/ViewItem"
 
-const HouseholdMemberForm = (props: { member: HouseholdMemberUpdate; type: string }) => {
+const HouseholdMemberForm = (props: {
+  member: HouseholdMemberUpdate
+  type: string
+  editMode?: boolean
+}) => {
   const { member, type } = props
   const router = useRouter()
+  const editMode = props.editMode !== false // undefined should default to true
```

Comment:
Was hoping to use a default prop of false here but this is def non-blocking!

Reply:
I think the weirdness is that the default state should actually be true, so only if editMode prop is present _and_ set specifically to false should it be set false.